app/legacy_lib/insert_bank_account.rb

```diff
@@ -52,7 +52,7 @@ def self.with_stripe(nonprofit, user, params)
         )
 
         NonprofitMailer.delay.new_bank_account_notification(bank_account)
-        return bank_account
+        bank_account
```

Comment:
Rails 6.1 deprecates calling a return directly inside a transaction. If you see a return being removed on some of these, it's because of that.